styleguide/components/11-base/forms/_forms.scss

```diff
@@ -79,43 +79,47 @@ $field-icon-padding: 2.5rem;
   }
 
   &::-webkit-input-placeholder { // sass-lint:disable-line no-vendor-prefixes
-    @include theme('color', 'color-secondary--lighten-1', 'fields-placeholder-color');
+    @include theme('color', 'color-tertiary--lighten-1', 'fields-placeholder-color');
     font-style: italic;
     font-weight: normal;
   }
 
   &:-moz-placeholder { // sass-lint:disable-line no-vendor-prefixes
-    @include theme('color', 'color-secondary--lighten-1', 'fields-placeholder-color');
+    @include theme('color', 'color-tertiary--lighten-1', 'fields-placeholder-color');
     font-style: italic;
     font-weight: normal;
   }
 
   &::-moz-placeholder { // sass-lint:disable-line no-vendor-prefixes
-    @include theme('color', 'color-secondary--lighten-1', 'fields-placeholder-color');
+    @include theme('color', 'color-tertiary--lighten-1', 'fields-placeholder-color');
     font-style: italic;
     font-weight: normal;
   }
 
   &:-ms-input-placeholder { // sass-lint:disable-line no-vendor-prefixes
-    @include theme('color', 'color-secondary--lighten-1', 'fields-placeholder-color');
+    @include theme('color', 'color-tertiary--lighten-1', 'fields-placeholder-color');
     font-style: italic;
     font-weight: normal;
   }
 
   &::placeholder {
-    @include theme('color', 'color-secondary--lighten-1', 'fields-placeholder-color');
+    @include theme('color', 'color-tertiary--lighten-1', 'fields-placeholder-color');
     font-style: italic;
     font-weight: normal;
   }
 
   display: block;
   width: 100%;
-  height: 48px; // @TODO: Change to rems based on the base font-size.
-  padding: 1rem;
+  height: 2.4rem;
+  padding: 0 .8rem;
   border: 1px solid;
   border-radius: border-radius('radius-1');
   font-family: $default-font-family;
   font-size: .7rem;
   font-weight: normal;
   appearance: none;
+
+  @include desktop {
+    font-size: .7rem;
```

Comment:
font-size .7rem? terwijl het mobile .7rem al is? :D mobile moest 1rem worden?

Reply:
Fixed.